drivers/mspi/mspi_stm32_ospi.c

```diff
@@ -0,0 +1,1854 @@
+/*
+ * Copyright (c) 2025 EXALT Technologies
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * **************************************************************************
+ * MSPI flash controller driver for stm32 series with multi-SPI peripherals
+ * This driver is based on the stm32Cube HAL OSPI driver
+ * with one mspi DTS NODE
+ * **************************************************************************
+ */
+#define DT_DRV_COMPAT st_stm32_ospi_controller
+
+#include <zephyr/cache.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/drivers/mspi.h>
+#include <zephyr/drivers/dma/dma_stm32.h>
+#include <zephyr/drivers/dma.h>
+#include <stm32_ll_dma.h>
```

Comment:
I would suggest to move this line below, next to the other `stm32_xxx.h` header inclusion.